src/Domain/Person.php

```diff
@@ -189,16 +206,6 @@ class Person
      */
     public ?string $completion_jwt = null;
 
-    /**
-     * Note this is defined here as part of the HTTP API, but is never actually set to true on the PHP object.
-     * {@see HasPasswordNormalizer}
-     *
-     * @deprecated because I keep forgetting not to use it from within PHP.
-     *
-     * @OA\Property()
-     */
-    public bool $has_password = false;
```

Comment:
afaik this php property was only here as a place to put the OA annotation - it was always false at runtime in the actual object. Moved the annotation to the class docblock.